staging/src/k8s.io/code-generator/pkg/util/build.go

```diff
@@ -50,3 +51,24 @@ func hasSubdir(root, dir string) (rel string, ok bool) {
 	// cut off root
 	return filepath.ToSlash(dir[len(root):]), true
 }
+
+// BoilerplatePath uses the boilerplate in k8s.io/code-generator.
+// If it does not exist, it uses the boilerplate in k8s.io/kubernetes.
+func BoilerplatePath() string {
+	boilerPlatePath := "k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt"
+	if ok, err := exists("k8s.io/code-generator/hack/boilerplate.go.txt"); err == nil && ok {
```

Comment:
This needs to use the full path `filepath.Join(args.DefaultSourceTree(),...` (will update)